python/ray/tests/test_autoscaler.py

```diff
@@ -484,11 +484,17 @@ def update_nodes(self):
         "ssh_user": "ubuntu",
         "ssh_private_key": os.devnull,
     },
-    "head_node": {
-        "TestProp": 1,
-    },
-    "worker_nodes": {
-        "TestProp": 2,
+    "available_node_types": {
```

Comment:
Somehow I though that the modifications needed for this file would be much worse.
Glad that this is all.

Reply:
it definitely took a hot sec to understand how to minimize changes here lol

Reply:
oh dear god i definitely didn't run this again after making some changes...